lxd/operations.go

```diff
@@ -1125,7 +1099,7 @@ func operationWebsocketGet(d *Daemon, r *http.Request) response.Response {
 	// First check if the query is for a local operation from this node
 	op, err := operations.OperationGetInternal(id)
 	if err == nil {
-		return &operationWebSocket{r, op}
+		return operations.OperationWebSocket(r, op)
```

Comment:
Could you add to the commit message that you are removing `operationWebSocket` because it is duplicating the `operationWebSocket` from the operations package? Thanks